pkgs/applications/misc/zathura/core/default.nix

```diff
@@ -9,12 +9,15 @@
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
-  pname = "zathura-core";
+  pname = "zathura";
```

Comment:
Then would make sense to also rename the `zathura_core` attribute to `zathura`.

Reply:
Upstream's project name for this is: https://pwmt.org/projects/zathura/download/ - `zathura`, not `zathura_core`.

Reply:
@doronbehar Here is a `zathura_core` attribute (inside a private scope):
https://github.com/NixOS/nixpkgs/blob/2cd18aba94dd500eb3693d4882f7d19b9caaf18b/pkgs/applications/misc/zathura/default.nix#L11